### DISCUSSION TITLE
Running indi-allsky on RPi-3B

### DISCUSSION THREAD

**Discussion by baryah** [aaronwmorris/indi-allsky#788] [General]
I had to increase the swap file size to 1024MB to make it run flawlessly RPi-3B, else Pi would hang and needed hard rebooting a couple of times.

**Comment by aaronwmorris** [comment]
Sorry, I guess I just missed this topic.

indi-allsky is a little heavy on memory.  1GB of memory is enough to operate the web interface and capture processes, however timelapse creation requires significant memory usage by itself.  ffmpeg needs about 512MB of memory by itself to build a 1920x1080 video and 1+GB for 4k video.

I always enabled the watchdog on my 1GB systems in case it hard locked due to memory issues.  The watchdog does a pretty good job of auto-rebooting the system.

https://github.com/aaronwmorris/indi-allsky/wiki/Raspberry-Pi-Watchdog

1GB of memory is really the absolute minimum amount of memory needed for indi-allsky.  2GB is strongly recommended.

**Comment by baryah** [comment]
Thanks
